### PR TITLE
recipe for version 2

### PR DESCRIPTION
**Pull request by mapo9**
Hi, we urgently need the version 2 of backports-datetime-fromisoformat, however currently only the [first version](https://github.com/conda-forge/backports-datetime-fromisoformat-feedstock) is available. I already tried this meta.yaml on the [staged_recipes](https://github.com/conda-forge/staged-recipes) repo where all tests passed. 
 

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mapo9** [comment]
@conda-forge-admin, please rerender

**Comment by mapo9** [comment]
Hi @ocefpaf,
all checks have passed for the v2 of the tool. 
Could you merge please?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,41 +1,39 @@
...
 {% set version = "2.0.0" %}
 
-
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/backports-datetime-fromisoformat-{{ version }}.tar.gz
```
Comment (ggabernet) [inline recipe/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{name}}-{{ version }}.tar.gz
```

Comment (ggabernet) [inline recipe/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{name}}-{{ version }}.tar.gz
```

**Review by ggabernet — COMMENTED** [review]
I just see a small correction here

**Comment by ocefpaf** [comment]
@mapo9 I'm OK merging this for now but this package is for a Python version we, conda-forge, no longer support. So that is probably the last binary we will issue for it.

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml:9]: @ggabernet this is nice to have but not mandatory. Also, we will probably archive this feedstock, so it is not worth working on it further. Thanks!

**Pull request merged**